core/flyout.js

```diff
@@ -657,11 +686,17 @@ Blockly.Flyout.prototype.getRect = function() {
   // the largest screen size.
   var BIG_NUM = 10000000;
   var x = Blockly.getSvgXY_(this.svgGroup_).x;
-  if (!this.RTL) {
+  if (!Blockly.RTL) {
```

Comment:
Blockly has no RTL property.  Workspaces have RTL (and potentially different workspaces might have different settings from each other).